molviewspec/app/api/examples.py

```diff
@@ -277,6 +277,23 @@ async def additional_properties_example() -> MVSResponse:
     return PlainTextResponse(builder.get_state())
 
 
+@router.get("/primitives")
+async def primitives_example() -> MVSResponse:
+    """
+    Any scene can be enriched with "geometric primitives" such as spheres, planes, etc. You can also define scenes using exclusively these nodes.
+    """
+    builder = create_builder()
+    # primitives are defined in "groups", you can add any number of primitives before invoking `options()` and setting
+    # shared properties such as color or transparency
+    builder.primitives().plane(point=(0, 1, 0), normal=(0, 1, 0)).options().label(text="Outer membrane").color(
```

Comment:
Did not quite get why the point and normal are defined by integers, not floats.

Reply:
[ae83a23](https://github.com/molstar/mol-view-spec/pull/36/commits/ae83a23a436b175c76cfd7112d9982f4ebd237df) changes the values to floats to make the typing more obvious.

Reply:
Ok, got it.